EventListener/JsonRequestTransformerListener.php

```diff
@@ -39,7 +39,7 @@ class JsonRequestTransformerListener
     /**
      * @param GetResponseEvent $event
      */
-    public function onKernelRequest(GetResponseEvent $event)
+    public function onKernelRequest(\Symfony\Component\HttpKernel\Event\RequestEvent $event)
```

Comment:
@rugbymauri Please replace the `GetResponseEvent` import (L32) instead, otherwise we are still referencing the old class.